protocol/chainlib/jsonRPC.go

```diff
@@ -291,7 +292,8 @@ func (apil *JsonRPCChainListener) Serve(ctx context.Context) {
 	app.Get("/:dappId/websocket", websocketCallbackWithDappID) // catching http://HOST:PORT/1/websocket requests.
 
 	app.Post("/:dappId/*", func(fiberCtx *fiber.Ctx) error {
-		apil.logger.LogStartTransaction("jsonRpc-http post")
+		endTx := apil.logger.LogStartTransaction("jsonRpc-http post")
+		defer endTx()
```

Comment:
nil check so we won;'t crash if the handler is dead

Reply:
add nil check inside the returned function from LogStartTransaction